### Eclipse Workspace Patch 1.0
#P org.eclipse.jdt.core.tests.compiler
Index: src/org/eclipse/jdt/core/tests/eval/DebugEvaluationSetup.java
===================================================================
RCS file: /cvsroot/eclipse/org.eclipse.jdt.core.tests.compiler/src/org/eclipse/jdt/core/tests/eval/DebugEvaluationSetup.java,v
retrieving revision 1.13
diff -u -r1.13 DebugEvaluationSetup.java
--- src/org/eclipse/jdt/core/tests/eval/DebugEvaluationSetup.java	23 Feb 2005 02:52:38 -0000	1.13
+++ src/org/eclipse/jdt/core/tests/eval/DebugEvaluationSetup.java	22 May 2007 10:21:06 -0000
@@ -153,7 +153,7 @@
 
 			// Create target
 			this.target = new TargetInterface();
-			this.target.connect("localhost", evalPort, 10000);
+			this.target.connect("localhost", evalPort, 30000);
 
 			// Create name environment
 			this.env = new FileSystem(Util.getJavaClassLibs(), new String[0], null);
Index: src/org/eclipse/jdt/core/tests/eval/SimpleTest.java
===================================================================
RCS file: /cvsroot/eclipse/org.eclipse.jdt.core.tests.compiler/src/org/eclipse/jdt/core/tests/eval/SimpleTest.java,v
retrieving revision 1.15
diff -u -r1.15 SimpleTest.java
--- src/org/eclipse/jdt/core/tests/eval/SimpleTest.java	29 Mar 2006 03:52:03 -0000	1.15
+++ src/org/eclipse/jdt/core/tests/eval/SimpleTest.java	22 May 2007 10:21:06 -0000
@@ -289,7 +289,7 @@
 
 	this.requestor = new Requestor();
 	this.target = new TargetInterface();
-	this.target.connect("localhost", evalPort, 10000);
+	this.target.connect("localhost", evalPort, 30000);
 	this.context = new EvaluationContext();
 }
 protected void stopEvaluationContext() {
Index: src/org/eclipse/jdt/core/tests/eval/EvaluationSetup.java
===================================================================
RCS file: /cvsroot/eclipse/org.eclipse.jdt.core.tests.compiler/src/org/eclipse/jdt/core/tests/eval/EvaluationSetup.java,v
retrieving revision 1.14
diff -u -r1.14 EvaluationSetup.java
--- src/org/eclipse/jdt/core/tests/eval/EvaluationSetup.java	10 May 2006 18:07:26 -0000	1.14
+++ src/org/eclipse/jdt/core/tests/eval/EvaluationSetup.java	22 May 2007 10:21:06 -0000
@@ -72,7 +72,7 @@
 
 			// Create target
 			this.target = new TargetInterface();
-			this.target.connect("localhost", evalPort, 10000);
+			this.target.connect("localhost", evalPort, 30000);
 
 			// Create name environment
 			this.env = new FileSystem(Util.getJavaClassLibs(), new String[0], null);
